### PR TITLE
#2357 added two actions 

### PR DESCRIPTION
**Pull request by feyzullah**
two actions;
* create-chat-invite-link
* export-chat-invite-link
note: export-chat-invite-link is needed for creating new invite links.

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/telegram_bot_api/actions/create-chat-invite-link/create-chat-invite-link.mjs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import telegramBotApi from "../../telegram_bot_api.app.mjs";
+
+export default {
+  key: "telegram_bot_api-create-chat-invite-link",
+  name: "Create Chat Invite Link",
+  description: "Use this method to create an additional invite link for a chat. The bot must be an administrator in the chat for this to work and must have the appropriate administrator rights. The link can be revoked using the method revokeChatInviteLink. Returns the new invite link as ChatInviteLink object. Note: Each administrator in a chat generates their own invite links. Bots can't use invite links generated by other administrators. If you want your bot to work with invite links, it will need to generate its own link using exportChatInviteLink or by calling the getChat method. If your bot needs to generate a new primary invite link replacing its previous one, use exportChatInviteLink again. [See the docs](https://core.telegram.org/bots/api#createchatinvitelink) for more information",
```
Comment (ctrlaltdylan) [inline components/telegram_bot_api/actions/create-chat-invite-link/create-chat-invite-link.mjs:6]: This description is going to be visible in the Pipedream dashboard for the user to read.

Try to keep this succinct to 1 sentence if you can. Here's an example: https://github.com/PipedreamHQ/pipedream/blob/750ff1d6cde94838d54a3bb48097d5d751b3e60c/components/telegram_bot_api/actions/edit-media-message/edit-media-message.mjs#L10

File: components/telegram_bot_api/actions/export-chat-invite-link/export-chat-invite-link.mjs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import telegramBotApi from "../../telegram_bot_api.app.mjs";
+
+export default {
+  key: "telegram_bot_api-export-chat-invite-link",
+  name: "Export Chat Invite Link",
+  description: "Use this method to generate a new primary invite link for a chat; any previously generated primary link is revoked. The bot must be an administrator in the chat for this to work and must have the appropriate administrator rights. Returns the new invite link as String on success.",
```
Comment (ctrlaltdylan) [inline components/telegram_bot_api/actions/export-chat-invite-link/export-chat-invite-link.mjs:6]: Same comment as above. Try to keep the description contained to 1 sentence.

**Review by ctrlaltdylan — DISMISSED** [review]
Otherwise looks good! Just 2 comments about the `description` field.

**Review by vellames-turing — APPROVED** [review]
Looks great, Approved!

**Pull request merged**